src/main/java/seedu/contax/logic/commands/DeleteTagCommand.java

```diff
@@ -16,9 +16,9 @@
 public class DeleteTagCommand extends Command {
     public static final String COMMAND_WORD = "deletetag";
     public static final String MESSAGE_USAGE = "`" + COMMAND_WORD
-            + "` : **Deletes the tag identified by tag name (case-insensitive).\n"
-            + "Deleting this tag will also remove contacts who have this tag.**\n"
-            + "Parameters: *INDEX (must be a positive integer)*\n"
+            + "` : **Deletes the tag identified by tag name (case-insensitive). "
+            + "Deleting this tag will also remove contacts who have this tag.**"
+            + "\nParameters: *INDEX (must be a positive integer)*\n"
             + "Example: `" + COMMAND_WORD + " 1`";
 
     public static final String MESSAGE_DELETE_TAG_SUCCESS = "Deleted Tag: %s";
```

Comment:
Similarly, there should be a break line after the colon here.

Reply:
thanks for the catch, updated in https://github.com/AY2122S2-CS2103-W17-1/tp/pull/248/commits/2b67c04764a53ad085721a659134752196fea472